2.401/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG JENKINS_VERSION=2.401.2
+
+FROM spryker/jenkins-boilerplate:${JENKINS_VERSION} as jenkins_boilerplate
+
+FROM jenkins/jenkins:${JENKINS_VERSION} as jenkins
```

Comment:
For me will be great to drop copy and leave a boilerplate inheritance. But we need to check how it increases the boilerplate image if we drop the alpine step from the image.

Reply:
In the past, I implemented it in that way, which resulted in an increased boilerplate image size from 150 MB to 540 MB, similar to the standard Jenkins image. Therefore, we made the decision to keep the boilerplate as it is and make a COPY on the spryker/docker-jenkins side.